src/ussd_popi_user_data.js

```diff
@@ -216,26 +225,12 @@ go.app = function() {
                 question: $('What would you like to do?'),
                 choices: [
                     new Choice('state_view', $('See my personal info')),
+                    new Choice('state_view_sms', $('Send my personal info as sms')),
```

Comment:
Yes. I ran out of options because I needed a paginated state for state_view, but paginated state only gives you the options for More and Exit, so I couldn't add the option to Send As Sms